### PR TITLE
Fix #409: Show the distribution and evolution views as tabs in dashboard

### LINKED ISSUE #409: Show the distribution and evolution views as tabs in dashboard
Currently it's not easy to see that we have two different dashboard views: a distribution view and an evolution view.
Effectively both views are very similar and behave like different view modes in a single dashboard.

We should more clearly highlight these and how to switch between them, probably using a "tab" metaphor in the UI. I did a simple mockup (note the removed button in the top bar and the added tabs before the probe selection):
dashboard mockup

### PR DESCRIPTION
**Pull request by Insiyaa**
Please review... Sorry for messing up:(

### CODE REVIEW

**Comment by Insiyaa** [comment]
Link: https://insiyaa.github.io/telemetry-dashboard/

**Comment by Sylvia23** [comment]
@Insiyaa, I think, you should `rebase` your repository by `fetching` and `merging` the `upstream` to resolve the `merge conflicts` in your GitHub PR.
@chutten , Is that right?

**Comment by Insiyaa** [comment]
@Sylvia23  Thanks for the suggestion :D

File: new-pipeline/dist.html [hunk 1]
```diff
@@ -24,12 +24,16 @@
             <form class="navbar-form inline pull-right">
                 <span class="permalink-control"></span>
                 <a href="https://github.com/mozilla/telemetry-dashboard/issues" class="btn btn-success" title="Report a bug in Telemetry dashboards" target="_blank"><i class="fa fa-bug"></i> Report bug</a>
-                <a href="./evo.html" class="btn btn-default" id="switch-views" title="Switch to the evolution view"><i class="fa fa-line-chart"></i> Evolution View</a>
+                <a href="/" class="btn btn-default btn-primary" title="Landing page for Telemetry dashboards"><i class="fa fa-home"></i> Dashboards Home</a>
```
Comment (chutten) [inline new-pipeline/dist.html]: "Dashboards Home" was removed by another PR. This was likely readded as part of your merge.

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (chutten) [inline _config.yml]: This appears unrelated? I don't know what this is.

File: new-pipeline/evo.html [hunk 3]
```diff
@@ -22,12 +22,16 @@
             <form class="navbar-form inline pull-right">
                 <span class="permalink-control"></span>
                 <a href="https://github.com/mozilla/telemetry-dashboard/issues" class="btn btn-success" title="Report a bug in Telemetry dashboards" target="_blank"><i class="fa fa-bug"></i> Report bug</a>
-                <a href="./dist.html" class="btn btn-default" id="switch-views" title="Switch to the distribution view"><i class="fa fa-bar-chart"></i> Distribution View</a>
+                <a href="/" class="btn btn-default btn-primary" title="Landing page for Telemetry dashboards"><i class="fa fa-home"></i> Dashboards Home</a>
```
Comment (chutten) [inline new-pipeline/evo.html]: Ditto here. You can just delete this button.

File: new-pipeline/style/dashboards.css [hunk 4]
```diff
@@ -128,7 +128,7 @@
   justify-content: center;
   align-items: center;
   text-align: center;
-  margin-bottom: 3em;
+  margin-bottom: 1em;
```
Comment (chutten) [inline new-pipeline/style/dashboards.css]: How do things look with `margin-bottom: 0;` ? That'll keep the plot at the same scroll index as it used to be.

Comment (Insiyaa) [reply new-pipeline/style/dashboards.css]: I actually can't check this out... The graphs don't load in my browser but they do in my phone. I wasn't able to judge by it...

File: new-pipeline/dist.html (hunk 1 above)
Comment (Insiyaa) [reply new-pipeline/dist.html]: Oh sorry

**Comment by Insiyaa** [comment]
Please review...

**Review by chutten — APPROVED** [review]
I'm not sure what would stop the graphs from loading on your computer. There's no special CSP headers or anything that would forbid something from loading, and your code is certainly not going to cause it.

A mystery for another time, I guess.

**Comment by chutten** [comment]
Thank you for your contribution!
